GoogleChromeLabs/ps-analysis-tool
# Feature: Extract information and show it on settings page for reporting bugs in the extension.

## Description

This PR aims to add a tab on the settings page that the user can use to copy and paste the issue description while reporting a bug in the extension.

## Relevant Technical Choices

- Chrome APIs are being used to get information such as installed extensions and current open tabs.
- Native functions are used to get the browser information.
- Once we get the information on the initial load of the settings page we store it in the state variable.
- This information stored in the state variable is then processed to make it readable.
- This information is sent to components to display in the information tab on the settings page.
- This information will now be used by users when they want to report a bug in the extension.

## Testing Instructions

- Checkout to the branch `fetch/extract-information-for-bugs`.
- Run `npm start` in the terminal.
- Right-click on the badge of the extension then select options
- In the settings page go to the information tab.
- you should now see all the information.

## Additional Information:



## Screenshot/Screencast



---



Fixes #92
